policyengine_uk/parameters/gov/dfe/extended_childcare_entitlement/disability_criteria.yaml

```diff
@@ -0,0 +1,18 @@
+description: The Department for Education exempts participants in these programs from work requirements of extended childcare entitlement. 
+metadata:
+  reference:
+    - title: The Childcare Regulations 2022 - Regulation 11A
+      href: https://www.legislation.gov.uk/uksi/2022/1134/regulation/11A
+  period: year
+  unit: program
+  label: Extended childcare entitlement disability criteria
+values:
+  2015-09-01:
+    - carers_allowance  # 11A(1)(a): Section 70 of the Social Security Contributions and Benefits Act 1992
+    - incapacity_benefit  # 11A(1)(f, g): Incapacity Benefit under Social Security Contributions and Benefits Act 1992
+    - is_disabled_for_benefits  # No direct match, but aligns with general disability-related eligibility
+    - is_enhanced_disabled_for_benefits  # No direct match, assumed to align with enhanced disability eligibility
+    - is_severely_disabled_for_benefits  # No direct match, assumed to align with severe disability eligibility
+    - esa_contrib  # 11A(1)(e): Employment and Support Allowance (ESA) under Welfare Reform Act 2007
+    - jsa_contrib  # Not explicitly listed in Regulation 11A, but likely relevant under other benefit schemes
```

Comment:
'likely relevant' I don't see why. And I'm not sure why an individual unemployment benefit would logically passport for childcare entitlement?